packages/drivers/odsp-urlResolver/src/urlResolver.ts

```diff
@@ -134,7 +136,11 @@ async function initializeFluidOfficeOrOneNote(
 
 	// Since we have the drive and item, only take the host ignore the rest
 	const siteUrl = decodedSite.slice(Math.max(0, storageType.length + 1));
-	const driveId = decodeURIComponent(siteDriveItemMatch[3]);
-	const itemId = decodeURIComponent(siteDriveItemMatch[4]);
+	// TODO Why are we non null asserting here
```

Comment:
Since multi-version api document is always enable now, there will be expecting one more array element for version number.